js/main.js

```diff
@@ -0,0 +1,230 @@
+
+let toggleInfoModal = false;
+const $infoModal = document.querySelector('#info-modal');
+const $infoExit = document.querySelector('.info-exit');
+
+let togglePlayModal = false;
+const $playModal = document.querySelector('#play-modal');
+const $playConfirm = document.querySelector('.play-confirm');
+const $playExit = document.querySelector('.play-exit');
+
+let toggleBookmarkModal = false;
+const $bookmarkModal = document.querySelector('#bookmark-modal');
+const $bookmarkConfirm = document.querySelector('.bookmark-confirm');
+const $bookmarkExit = document.querySelector('.bookmark-exit');
+
+$ulList.addEventListener('click', function (event) {
+  if (event.target.tagName !== 'I') {
+    return;
+  }
+
+  const $animeLi = event.target.closest('li');
+  const infoImg = document.querySelector('#info-img');
+  const infoTitle = document.querySelector('#info-title');
+  const synopsis = document.querySelector('#synopsis');
+
+  if (event.target.classList.contains('info')) {
+    for (let i = 0; i < data.list.length; i++) {
+      if (data.list[i].mal_id === Number($animeLi.dataset.malId)) {
+        infoImg.setAttribute('src', data.list[i].images.jpg.image_url);
+
+        if (data.list[i].title_english === null) {
+          infoTitle.textContent = data.list[i].title;
+        } else infoTitle.textContent = data.list[i].title_english;
+
+        synopsis.textContent = data.list[i].synopsis;
+      }
+    }
+    toggleInfoModal = !toggleInfoModal;
+    if (toggleInfoModal === true) {
+      $infoModal.classList.remove('hidden');
+    }
+  }
+
+  if (event.target.classList.contains('play')) {
+    togglePlayModal = !togglePlayModal;
+    if (togglePlayModal === true) {
+      $playModal.classList.remove('hidden');
+    }
+  }
+
+  if (event.target.classList.contains('bookmark')) {
+    toggleBookmarkModal = !toggleBookmarkModal;
+    if (toggleBookmarkModal === true) {
+      $bookmarkModal.classList.remove('hidden');
+    }
+  }
+});
+
+$infoExit.addEventListener('click', function (event) {
+  toggleInfoModal = !toggleInfoModal;
+  if (toggleInfoModal === false) {
+    $infoModal.classList.add('hidden');
+  }
+});
+
+$playExit.addEventListener('click', function (event) {
+  togglePlayModal = !togglePlayModal;
+  if (togglePlayModal === false) {
+    $playModal.classList.add('hidden');
+  }
+});
+
+$bookmarkExit.addEventListener('click', function (event) {
+  toggleBookmarkModal = !toggleBookmarkModal;
+  if (toggleBookmarkModal === false) {
+    $bookmarkModal.classList.add('hidden');
+  }
+});
+
+const $entryList = document.querySelector('#entry-list');
+
+const xhr = new XMLHttpRequest();
+xhr.open('GET', 'https://api.jikan.moe/v4/seasons/now?page=1');
+xhr.responseType = 'json';
+xhr.addEventListener('load', function () {
+  data.list = xhr.response.data;
+  for (let i = 0; i < xhr.response.data.length; i++) {
+    const animeEntry = renderAnime(xhr.response.data[i]);
+    $entryList.appendChild(animeEntry);
+  }
+});
+xhr.send();
```

Comment:
I would put this inside a function rather than all out globally. Then you can call the function in a `DOMContentLoaded` event listener